scopeql/src/repl/entrypoint.rs

```diff
@@ -80,6 +80,7 @@ pub fn entrypoint(config: Config) {
     let hinter = DefaultHinter::default().with_style(Style::new().fg(Color::DarkGray));
 
     let mut state = Reedline::create()
+        .use_bracketed_paste(true)
```

Comment:
https://docs.rs/reedline/latest/reedline/struct.Reedline.html#method.use_bracketed_paste

Otherwise paste code like:

```
VALUES
('{ 
    "date" : "2017-04-28", 
}');
```

would fail because tokenzier would fail to recognize `('{` partially.